iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/AbstractIdentifiableAdderAdapter.java

```diff
@@ -7,13 +7,14 @@
 package com.powsybl.iidm.mergingview;
 
 import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Identifiable;
 import com.powsybl.iidm.network.IdentifiableAdder;
 import com.powsybl.iidm.network.util.Identifiables;
 
 /**
  * @author Thomas Adam <tadam at silicom.fr>
  */
-abstract class AbstractIdentifiableAdderAdapter<I extends IdentifiableAdder<I>> extends AbstractAdapter<I> implements IdentifiableAdder<I> {
+abstract class AbstractIdentifiableAdderAdapter<T extends Identifiable<? super T>, I extends IdentifiableAdder<T, I>> extends AbstractAdapter<I> implements IdentifiableAdder<T, I> {
```

Comment:
Same question as in `BranchAdder`

Reply:
It's again because of `TieLine`, which makes BranchAdder have the same kind of parameter, and as BranchAdder extends AbstractIdentifiable we need the same here.

Reply:
After the merge I needed the same for AbstractSimpleIdentifiableXml